### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by bsipocz**
Fixing typos that got merged since #3462.

### CODE REVIEW

**Comment by adrn** [comment]
Impressive :+1:

**Comment by astrofrog** [comment]
How do we keep missing all these??! Thanks @bsipocz!

File: astropy/units/function/core.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ class FunctionUnitBase(object):
     function_unit :  `~astropy.units.Unit` or `string`
         By default, the same as the function unit set by the subclass.
     """
-    # vvvvv the following four need to be set by subclasses
```
Comment (mhvk) [inline astropy/units/function/core.py]: Hmm, those "vvvvv" were there on purpose; some are matched by "^^^^^" a bit lower down. In principle, I don't mind them being removed if that helps the spell checking (which is very much appreciated!), but do remove the "^^^^" as well!

Comment (bsipocz) [inline astropy/units/function/core.py]: Sorry, I haven't noticed those. Somehow assumed these just remained here as placeholders (I do similar things with easily grep-able strings but with the aim to remove them all). Will revoke these changes.

**Comment by Cadair** [comment]
Are you using something automated to help you out with this? I dread to think what would happen if I ran a spell checker on the whole SunPy code base!

**Comment by bsipocz** [comment]
@Cadair -  I should have come up with something by now, but sadly I don't have an automated solution. I just check the diffs since a previous typo search sometimes when I'm at a useless state (not tired enough to be able to sleep, but too tired to do anything useful...).
I can go through SunPy in the coming weeks if you like.

**Comment by bsipocz** [comment]
btw a disclaimer: the spell checker suggested for many composite (or composite-like) words to be written separately or with a hyphen. I never feel comfortable to decide which way is the right one, so always went with the original version.

**Comment by embray** [comment]
:+1:

**Pull request merged**